lnclient/ldk/ldk.go

```diff
@@ -999,9 +999,10 @@ func (ls *LDKService) GetOnchainBalance(ctx context.Context) (*lnclient.OnchainB
 }
 
 func (ls *LDKService) RedeemOnchainFunds(ctx context.Context, toAddress string) (string, error) {
-	txId, err := ls.node.OnchainPayment().SendAllToAddress(toAddress)
+	spendableBalance := ls.node.ListBalances().SpendableOnchainBalanceSats
+	txId, err := ls.node.OnchainPayment().SendToAddress(toAddress, spendableBalance)
```

Comment:
Maybe would be nice to add a TODO above to try detect the transaction fee and subtract that from the spendable balance